.github/workflows/build.yml

```
@@ -0,0 +1,86 @@
+        run: mvn clean install
+
+      # Upload the JAR as an artifact using v4
+      - name: Upload JAR as artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: bus-booking-app
+          path: target/simple-parcel-service-app-1.0-SNAPSHOT.jar
+
+      # Run the Spring Boot application
+      - name: Run Spring Boot App
+        run: mvn spring-boot:run & 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)